packages/preview/source/cli.ts

```diff
@@ -27,6 +27,11 @@ Options:
 }
 
 export async function run(argv: any) {
+  if (argv._[0] && !/^(build|serve)$/i.test(argv._[0])) {
```

Comment:
Could we use a destructuring pattern to extract params here?
```suggestion
  // insert serve option between path and preview
  const [rootPath] = argv._;
  if (rootPath && !/^(build|serve)$/i.test(rootPath)) {
```